doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -182,7 +184,7 @@ This section provides detailed lists of changes by :ref:`application <applicatio
 Machine learning
 ----------------
 
-|no_changes_yet_note|
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
```

Comment:
```suggestion
* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```

Reply:
@peknis This is about the board target, I'd keep it as is.